esx_service/utils/kvESX.py

```diff
@@ -166,6 +168,14 @@ def delete(volpath):
     return True
 
 
+def align_str(kv_str):
+   # Align a given KV meta-data string to the next 4K boundary.
+   aligned_len = ((len(kv_str) + (KV_ALIGN - 1)) / KV_ALIGN) * KV_ALIGN - 1
+   padded_str = '{:<{width}}'.format(kv_str, width=aligned_len)
+
+   # Add a trailing newline to bring up string length to a 4K boundary.
+   return '{0}{1}'.format(padded_str, '\n')
```

Comment:
why not use default (space) for padding ? Something like 
`('{:<%d}` % round_up(len(kv_str, 4096))).format(kv_str)`
(assuming `def round_up(n, block): return (( n + block - 1 ) / block ) * block`